platform/javascript/detect.py

```diff
@@ -20,6 +22,14 @@ def get_opts():
     from SCons.Variables import BoolVariable
 
     return [
+        ("initial_memory", "Initial WASM memory (in MiB)", 16),
+        BoolVariable("use_assertions", "Use emscripten runtime assertions", False),
+        BoolVariable("use_thinlto", "Use ThinLTO", False),
+        BoolVariable("use_ubsan", "Use LLVM/GCC compiler undefined behavior sanitizer (UBSAN)", False),
+        BoolVariable("use_asan", "Use LLVM/GCC compiler address sanitizer (ASAN)", False),
+        BoolVariable("use_lsan", "Use LLVM/GCC compiler leak sanitizer (LSAN)", False),
+        BoolVariable("use_safe_heap", "Use emscripten SAFE_HEAP sanitizer", False),
+
```

Comment:
`black` doesn't like an empty line in the array.

Reply:
I forgot to copy that change :+1: